pkg/reconciler/ingress/controller.go

```diff
@@ -179,3 +182,10 @@ func combineFunc(functions ...func(interface{})) func(interface{}) {
 		}
 	}
 }
+
+func getSecretInformer(ctx context.Context) v1.SecretInformer {
+	if pkg.ShouldFilterByCertificateUID() {
```

Comment:
I think I'd prefer we look up the LabelKey from the context - then the label could be configured

https://github.com/knative/pkg/blob/2d8305b2e59ac921cbb125e83a9c852b895124c5/client/injection/kube/informers/factory/filtered/filtered_factory.go#L41-L45